modules/installation-user-infra-machines-pxe.adoc

```diff
@@ -146,8 +176,8 @@ For example, to use DHCP on a NIC that is named `eno1`, set `ip=eno1:dhcp`.
 ====
 This configuration does not enable serial console access on machines with a graphical console.  To configure a different console, add one or more `console=` arguments to the `kernel` line.  For example, add `console=tty0 console=ttyS0` to set the first PC serial port as the primary console and the graphical console as a secondary console.  For more information, see link:https://access.redhat.com/articles/7212[How does one set up a serial terminal and/or console in Red Hat Enterprise Linux?].
 ====
+endif::only-pxe[]
 
-. Monitor the progress of the {op-system} installation on the console of the machine.
 +
 [IMPORTANT]
```

Comment:
Removing the step before this `[IMPORTANT]` block (while keeping the `+` continuation) has sorta orphaned the admonition. Should it be attached to the previous step? Or turned instead into an actual step itself?